iscsi/initiator.go

```diff
@@ -151,16 +146,16 @@ func LogoutTarget(ip, target string, ne *util.NamespaceExecutor) error {
 	if ip != "" {
 		opts = append(opts, "-p", ip)
 	}
-	_, err := ne.Execute(iscsiBinary, opts)
+	_, err := nsexec.Execute(iscsiBinary, opts, lhtypes.ExecuteDefaultTimeout)
 	return err
 }
 
-func GetDevice(ip, target string, lun int, ne *util.NamespaceExecutor) (*util.KernelDevice, error) {
+func GetDevice(ip, target string, lun int, nsexec *lhns.Executor) (*lhtypes.BlockDeviceInfo, error) {
```

Comment:
We need to fix the caller side if they are using `util.KernelDevice` now. I believe this has been tackled in other Talos PRs.